### PR TITLE
[luci-interpreter] Enable broadcast of Prelu

### PR DESCRIPTION
**Pull request by jinevening**
This enables broadcast in Prelu kernel

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

File: compiler/luci-interpreter/src/kernels/Prelu.test.cpp [hunk 1]
```diff
@@ -107,6 +132,58 @@ TEST(PreluTest, Uint8Simple)
...
+  kernel.execute();
+
+  EXPECT_THAT(dequantize(extractTensorData<uint8_t>(output_tensor), output_tensor.scale(),
+                         output_tensor.zero_point()),
+              ElementsAreArray(ArrayFloatNear(ref_output_data, kQuantizedTolerance)));
+  EXPECT_THAT(extractTensorShape(output_tensor), ::testing::ElementsAreArray({1, 2, 2, 3}));
+  EXPECT_THAT(extractTensorData<uint8_t>(output_tensor),
+              ::testing::ElementsAreArray(ref_quant_output_data));
+
+  SUCCEED();
```
Comment (s-barannikov) [inline compiler/luci-interpreter/src/kernels/Prelu.test.cpp:184]: Looks unnecessary :)

**Review by s-barannikov — DISMISSED** [review]

File: compiler/luci-interpreter/src/kernels/Prelu.test.cpp (hunk 1 above)
Comment (jinevening) [reply compiler/luci-interpreter/src/kernels/Prelu.test.cpp]: I've removed it. Thanks.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by s-barannikov — APPROVED** [review]
LGTM

**Pull request merged**